install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script goal is to generate the Ansible hosts and ansible.cfg files
```

Comment:
and to run the ansible script as well, right?

Reply:
Yes. Updated